okex-client/tests/client.rs

```diff
@@ -136,3 +151,83 @@ async fn transfer_state() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn place_order() -> anyhow::Result<()> {
+    if let (Ok(_), Ok(_), Ok(_)) = (
```

Comment:
This makes no sense - just have `demo_okex_client` return `Option<OkexClient>`

Reply:
I think that it is better to have `demo_okex_client` return `OkexClient` and to set the environment variables on github. This way,  the affected tests run and pass only when the variables have been set. I got rid of the `if` block.